jenkins/deploy/Jenkinsfile

```diff
@@ -0,0 +1,96 @@
+pipeline {
+    agent any
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: "5"))
+        timeout time: 60, unit: "MINUTES"
+    }
+
+    stages {
+        stage("Build") {
```

Comment:
If there are plans to make `Build` stage out of this Jenkinsfile replacing the Jenkins `iis_build` config in future:

http://ci.ceon.pl/view/IIS/job/iis_build

then we should probably keep the sonar in the loop (as defined within the M3 goals and options): 
```
clean package -DskipITs -Djava.net.preferIPv4Stack=true $SONAR_MAVEN_GOAL -Dsonar.host.url=$SONAR_HOST_URL
```

Reply:
Yes, that's a good remark, I've copied maven command blindly and forgot the check the configuration of `iis_build` job. I'll make maven run the same as `iis_build`. One note - this comment is attached to deploy Jenkinsfile but I think this change should be applied to 'test' Jenkinsfile that is the one that will be used to create Jenkins job for running unit tests and pushing the status to GitHub. Deployment build would only execute unit tests.